app/views/application/_header.slim

```diff
@@ -0,0 +1,9 @@
+header
+	h1 = link_to "Taskopolis", root_path
+	nav
+		ul
+			- if user_signed_in?
+				li = link_to "Log out", destroy_user_session_path, method: :delete
+			- else
+				li = link_to "Register", new_user_registration_path
+				li = link_to "Log in", new_user_session_path
```

Comment:
I'm using tabs, but no objections to changing.

No objection to 2 spaces! I'll configure ST to accommodate.